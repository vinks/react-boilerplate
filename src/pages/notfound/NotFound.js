import React from 'react'
import styles from 'pages/notfound/NotFound.css'

function NotFound() {
  return (
    <div className={styles.root}>
      404 Not Found
    </div>
  )
}

export default NotFound
