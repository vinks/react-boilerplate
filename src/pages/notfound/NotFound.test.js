/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'
import NotFound from 'pages/notfound/NotFound'

describe('pages/notfound/NotFound', () => {
  it('renders correctly', () => {
    const component = renderer.create(<NotFound />)

    const tree = component.toJSON()

    expect(tree).toMatchSnapshot()
  })
})
