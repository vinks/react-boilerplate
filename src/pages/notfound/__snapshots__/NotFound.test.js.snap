// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/notfound/NotFound renders correctly 1`] = `
<div
  className="root"
>
  404 Not Found
</div>
`;
